draft-ietf-tls-esni.md

```diff
@@ -730,7 +739,9 @@ added behavior:
   to ECH PSKs. ECH PSKs offered by the client are associated with the ECH
   name. The server was unable to decrypt then ECH name, so it should not resume
   them when using the plaintext SNI name. This restriction allows a client to
-  reject resumptions in {{auth-public-name}}.
+  reject resumptions in {{auth-public-name}}. [[TODO: Decide if this rule is
```

Comment:
+1 -- let's drop it!

Reply:
I'm revising it to: "If offered, the server MUST ignore the "pre_shared_key" extension sent in the ClientHello."